[2025-02-27T00:00:02.055+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: yfinance_ingestion_dag.fetch_data scheduled__2025-02-26T00:00:00+00:00 [queued]>
[2025-02-27T00:00:02.063+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: yfinance_ingestion_dag.fetch_data scheduled__2025-02-26T00:00:00+00:00 [queued]>
[2025-02-27T00:00:02.064+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2025-02-27T00:00:02.078+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): fetch_data> on 2025-02-26 00:00:00+00:00
[2025-02-27T00:00:02.084+0000] {standard_task_runner.py:57} INFO - Started process 1507 to run task
[2025-02-27T00:00:02.087+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'yfinance_ingestion_dag', 'fetch_data', 'scheduled__2025-02-26T00:00:00+00:00', '--job-id', '454', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmprif1z1ao']
[2025-02-27T00:00:02.090+0000] {standard_task_runner.py:85} INFO - Job 454: Subtask fetch_data
[2025-02-27T00:00:02.138+0000] {task_command.py:415} INFO - Running <TaskInstance: yfinance_ingestion_dag.fetch_data scheduled__2025-02-26T00:00:00+00:00 [running]> on host 09ff8bd65149
[2025-02-27T00:00:02.223+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='etudiant' AIRFLOW_CTX_DAG_ID='yfinance_ingestion_dag' AIRFLOW_CTX_TASK_ID='fetch_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-26T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-26T00:00:00+00:00'
[2025-02-27T00:00:02.261+0000] {history.py:247} ERROR - $BTC-USD: possibly delisted; no price data found  (1h 2025-02-26 00:00:02.225702 -> 2025-02-27 00:00:02.225702)
[2025-02-27T00:00:02.262+0000] {logging_mixin.py:151} INFO - Aucune donnée pour BTC-USD
[2025-02-27T00:00:02.271+0000] {history.py:247} ERROR - $ETH-USD: possibly delisted; no price data found  (1h 2025-02-26 00:00:02.225702 -> 2025-02-27 00:00:02.225702)
[2025-02-27T00:00:02.272+0000] {logging_mixin.py:151} INFO - Aucune donnée pour ETH-USD
[2025-02-27T00:00:02.281+0000] {history.py:247} ERROR - $SOL-USD: possibly delisted; no price data found  (1h 2025-02-26 00:00:02.225702 -> 2025-02-27 00:00:02.225702)
[2025-02-27T00:00:02.282+0000] {logging_mixin.py:151} INFO - Aucune donnée pour SOL-USD
[2025-02-27T00:00:02.283+0000] {taskinstance.py:1943} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/pipeline.py", line 64, in fetch_crypto_data
    raise ValueError("Aucune donnée récupérée depuis yfinance")
ValueError: Aucune donnée récupérée depuis yfinance
[2025-02-27T00:00:02.298+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=yfinance_ingestion_dag, task_id=fetch_data, execution_date=20250226T000000, start_date=20250227T000002, end_date=20250227T000002
[2025-02-27T00:00:02.311+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 454 for task fetch_data (Aucune donnée récupérée depuis yfinance; 1507)
[2025-02-27T00:00:02.340+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2025-02-27T00:00:02.363+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
